ngraph/frontend/onnx/frontend/include/onnx_import/core/null_node.hpp

```diff
@@ -31,7 +31,7 @@ namespace ngraph
         ///
         /// More:
         /// https://github.com/onnx/onnx/blob/master/docs/IR.md#optional-inputs-and-outputs
-        class NullNode : public ngraph::Node
+        class ONNX_IMPORTER_API NullNode : public ngraph::Node
```

Comment:
why is it migrated from private API?

Reply:
I guess it should be a part of private API

Reply:
I think it should be public. It was overlooked before, but we need that for custom onnx ops mechanism. NullNode is the way to represent missing optional input or output and I think we should allow users to use it like we do e.g. in batch norm https://github.com/openvinotoolkit/openvino/blob/master/ngraph/frontend/onnx/onnx_import/src/op/batch_norm.cpp#L38

Reply:
Before the NullNode was hidden and it was a private API.
Looks like in the case of optional inputs it can be needed for majority frameworks.
@slyalin Could you comment?

Reply:
@slyalin ping

I just want to know can this operation be used by other frontends? Should we move it to common part?